packages/webapp/pages/onboarding.tsx

```diff
@@ -309,14 +330,21 @@ export function OnboardPage(): ReactElement {
         showOnboardingPage={showOnboardingPage}
         setAuth={setAuth}
         customActionName={customActionName}
-        onClickCreateFeed={onClickCreateFeed}
+        onClick={
+          activeScreen === OnboardingStep.AndroidApp
+            ? onCompleteOnboarding
+            : onClickCreateFeed
+        }
         activeScreen={activeScreen}
       />
       <div
         className={classNames(
-          'flex w-full flex-grow flex-col flex-wrap justify-center px-4 tablet:flex-row tablet:gap-10 tablet:px-6',
+          'flex w-full flex-col flex-wrap justify-center px-4 tablet:flex-row tablet:gap-10 tablet:px-6',
           activeScreen === OnboardingStep.Intro && wrapperMaxWidth,
-          !isAuthenticating && 'mt-7.5 flex-1 content-center',
+          !isAuthenticating &&
+            activeScreen !== OnboardingStep.AndroidApp &&
+            'mt-7.5 flex-1 content-center',
+          activeScreen !== OnboardingStep.AndroidApp && 'flex-1 flex-grow',
```

Comment:
Why are these changes needed, it feels like it could affect some stuff and i don't think it should be needed, but maybe you can tell me what didn't work?

Reply:
@rebelchris There's a small error on my part since flex-1 shouldn't be a part of the check. 

However, I found a solution that doesn't need to shuffle around the already existing classes.